app/code/core/Mage/Core/etc/system.xml

```diff
@@ -1058,6 +1058,25 @@
             <show_in_website>0</show_in_website>
             <show_in_store>0</show_in_store>
             <groups>
+                <design translate="label">
+                    <label>Admin Theme</label>
+                    <frontend_type>text</frontend_type>
+                    <sort_order>5</sort_order>
+                    <show_in_default>1</show_in_default>
+                    <show_in_website>0</show_in_website>
+                    <show_in_store>0</show_in_store>
+                    <fields>
+                        <use_legacy_theme translate="label comment">
+                            <label>Enable legacy Admin Theme</label>
+                            <comment>Use legacy theme from Magento CE</comment>
+                            <frontend_type>select</frontend_type>
+                            <source_model>adminhtml/system_config_source_yesno</source_model>
+                            <sort_order>1</sort_order>
+                            <show_in_default>1</show_in_default>
```

Comment:
can I suggest to add also show_in_website and show_in_store even if the parent is not viewable in website and store?

Reply:
> LGT. Found just some missing header-icons. 💯

What means LGT? Could you name where the missing icons should be? I only see some missing icons that were missing in a vanilla magento as well. Like on "Catalog Price Rules"

> And maybe check spacing for dropdowns .... Reports -> Customers -> New customers .... "Show by Day" looks like "Show by Dav".

Fixed!

Thanks for testing!

Reply:
just commited your suggestion!